config.json

```diff
@@ -467,6 +467,15 @@
       ]
     },
     {
+      "slug": "circular-buffer",
+      "difficulty": 8,
+      "topics": [
+        "Queues",
+        "Classes",
+        "Exceptions"
```

Comment:
It looks like we're [going to be adding topics for every exercise soon](https://github.com/exercism/discussions/issues/159), so I'm a +1 to what @FridaTveit suggested :)

>In order to prepare your track for the new track progression model, you'll need to do the following:
>
>- **add topics to each exercise**. We have a list of topics that you can use for inspiration, but this list is not exhaustive. The topics you choose should make sense for your language.
>- add a rough guess at the difficulty of each exercise.
>- choose 10-20 exercises to mark as core
>- give the core exercises an order
>- for all other exercises, mark them as `unlocked_by` one of the core exercises (identified by the slug)

While we shouldn't embark on the above en masse just yet, adding topics is therefore good future-proofing!